WordPress/Classes/ViewRelated/Me/MeViewController.swift

```diff
@@ -74,7 +84,17 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
         // My guess is the table view adjusts the height of the first section
         // based on if there's a header or not.
         tableView.tableHeaderView = account.map { headerViewForAccount($0) }
+
+        // After we've reloaded the view model, we'll either reselect the currently
+        // selected table row, or we'll select the first row if the split view
+        // is collapsed.
+        var selectedIndexPath = tableView.indexPathForSelectedRow
+        if selectedIndexPath == nil && !splitViewControllerIsCollapsed {
+            selectedIndexPath = NSIndexPath(forRow: 0, inSection: 0)
+        }
+
         handler.viewModel = tableViewModel(loggedIn, helpshiftBadgeCount: badgeCount)
+        tableView.selectRowAtIndexPath(selectedIndexPath, animated: false, scrollPosition: .None)
```

Comment:
Should we maybe only call this code when a `UISplitViewController` is actually initialized?

This might read a bit confusing if someone comes across needing to maintain the code, and also understanding when or why there may be a split view, or if they still need to call `selectRowAtIndexPath`.